aspnetcore/includes/add-EF-NuGet-SQLite-CLI-5.md

```diff
@@ -0,0 +1,23 @@
+Run the following .NET Core CLI commands:
+
+```dotnetcli
+dotnet tool install --global dotnet-ef -v 5.0.0-rc.2.20475.6
```

Comment:
Yes, that is on my sched to remove them.  The 3.x and 2.x versions in the docs have the package versions spelled out.

Reply:
@scottaddie ( cc @Rick-Anderson )
Follow up: This was on my list for today, but I ended up spending most of the day on testing/review the MVC series.  However Rick updated them a few hours ago..   There is a mistake of using - v instead of version on the install that I must have added in there at some point.  I had retested this set so many times, but I must have missed that at some point after - v was accidently put in.   Thanks Rick for the update on this include.